lib/matplotlib/cm.py

```diff
@@ -23,10 +23,20 @@
 import matplotlib as mpl
 from matplotlib import _api, colors, cbook
 from matplotlib._cm import datad
+import functools
```

Comment:
This belongs with `collections.abc` in the standard library import block.

Reply:
Anybody is welcome to fix this and force-push. I won't be able to do it myself for a couple more days.

Reply:
So playing with the new codespaces, review comments on branches that have PRs are inline which is pretty nice!

Reply:
but the "mpl contributors can push to my PR branch" ACL does not seem to be forwarded yet.  Oh well.